aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md

```diff
@@ -68,17 +68,17 @@ Start Visual Studio. From the **File** menu, click **New Project**.
 
 In the **New Project** dialog box, expand **Visual C#**. Under **Installed Templates**, select **Cloud** and then select **Windows Azure Cloud Service**. Keep the default .NET Framework 4.5. Name the application ChatService and click **OK**.
 
-![Window titled 'New Project'. 'Cloud' is selected and "ChatService" is typed under 'Name'.](scaleout-with-windows-azure-service-bus/_static/image4.png)
+![Screenshot of window with Cloud selected and ChatService typed under Name.](scaleout-with-windows-azure-service-bus/_static/image4.png)
 
 In the **New Windows Azure Cloud Service** dialog, select ASP.NET Web Role. Click the right-arrow button (**&gt;**) to add the role to your solution.
 
 Hover the mouse over the new role, so the pencil icon visible. Click this icon to rename the role. Name the role "SignalRChat" and click **OK**.
 
-![Window titled 'New Windows Azure Cloud Service'. ASP.NET Web Role in the box on the right is labeled 'SignalRChat'.](scaleout-with-windows-azure-service-bus/_static/image5.png)
+![Screenshot of window titled New Windows Azure Cloud Service and Signal R Chat is typed above A S P dot NET Web Role.](scaleout-with-windows-azure-service-bus/_static/image5.png)
```

Comment:
How about, "Screenshot that shows the New Windows Azure Cloud Service dialog box. Signal R Chat is typed above A S P dot NET Web Role."